apigee_edge.module

```diff
@@ -1463,6 +1464,18 @@ function apigee_edge_get_my_developer_apps_title(?UserInterface $user = NULL): T
  * Implements hook_cache_flush().
  */
 function apigee_edge_cache_flush() {
-  // Clear token data.
-  \Drupal::service('apigee_edge.authentication.oauth_token_storage')->removeToken();
+  // Clear OAuth token storage and enforce re-authentication to Apigee Edge.
+  $active_key_id = \Drupal::config('apigee_edge.auth')->get('active_key');
+  if ($active_key_id) {
```

Comment:
For start, this method calls `removeToken()` not `removeTokenFile()`. There is a huge difference between them.

To answer your question, there are multiple reasons why we have to do this sanity check:
* The module should not always trigger an Oauth token cache clear.
  * Not when the module has just installed and 100% it has not been configured yet. Calling `removeToken()` creates the token file it has not existed yet. This file should always exists if Oauth authentication is in use.
 * The private file system in Drupal is not configured by default. So if cache clear tries to clear the token file if the private file system is not configured (when the module is installed for example or has not been configured yet) because "private://" scheme in Drupal only exist if private filesystem path is configured. That is also going to trigger error.
  * Not when the HTTP authentication method is in use - it should not create an empty token storage file, etc.
  * ...

So basically we are doing a shortcut by not doing a check in the `hook_uninstall()` to see whether private filesystem was configured or not and we just assume it was.